backend/app/minio/config.py

```
@@ -14,8 +13,13 @@
 
 BUCKET_NAME = "media"
 
-if not MINIO_CLIENT.bucket_exists(BUCKET_NAME):
-    MINIO_CLIENT.make_bucket(BUCKET_NAME)
-    print(f"Bucket '{BUCKET_NAME}' created successfully.")
-else:
-    print(f"Bucket '{BUCKET_NAME}' already exists.")
+def initialize_minio_bucket():
```

Ruff (0.8.2)

Missing return type annotation for public function `initialize_minio_bucket`

Add return type annotation: `None`

(ANN201)